project/src/main/java/com/google/sps/data/PlacesFetcher.java

```diff
@@ -39,20 +46,22 @@
 
     /** In this radius around "LOCATION" places will be searched. */
     private static final int SEARCH_RADIUS = 5000;
-    // TODO(M1): check at least 10 results, and if less extend radius
 
     /** The entry point for a Google GEO API request. */
     private static final GeoApiContext CONTEXT = new GeoApiContext.Builder()
         .apiKey(System.getenv("API_KEY"))
         .build();
 
+    /** A mapping between cuisines and text search words. */
+    private static final Map<String, String[]> CUISINE_TO_SEARCH_WORDS_MAP = getCuisinesMap();
```

Comment:
Use ImmutableHashMap.

Reply:
I changed the Map to immutableMap as i didn't find an ImmutableHashMap type.

Reply:
Changes aren't visible..